src/cloudai/__main__.py

```diff
@@ -253,11 +254,39 @@ def handle_generate_report(test_scenario: TestScenario, output_dir: Path) -> Non
     logging.info("Report generation completed.")
 
 
+def handle_verify_systems(root: Path) -> int:
+    if not root.exists():
+        logging.error(f"Tests directory {root} does not exist.")
+        return 1
+
+    test_tomls = [root]
+    if root.is_dir():
+        test_tomls = list(root.glob("*.toml"))
+        if not test_tomls:
+            logging.error(f"No test tomls found in {root}")
+            return 1
+
+    rc = 0
+    for test_toml in test_tomls:
+        logging.info(f"Verifying {test_toml}...")
+        try:
+            Parser.parse_system(test_toml)
+        except Exception:
+            rc = 1
+            break
+
```

Comment:
Do we print proper error messages upon failure? Based on my experience with CloudAI users, it is always better to provide more descriptive information, warning, and error messages for debugging purposes.

Reply:
Yes, here is how it looks like:
```bash
cloudai --mode verify-systems --tests-dir s --test-templates-dir d --system-config conf/common/system/example_slurm_cluster.toml
[INFO] Verifying conf/common/system/example_slurm_cluster.toml...
[ERROR] Validation error: {'type': 'missing', 'loc': ('name',), 'msg': 'Field required', 'input': {'scheduler': 'slurm', 'install_path': './install', 'output_path': './results', 'default_partition': 'partition_1', 'mpi': 'pmix', 'gpus_per_node': 8, 'ntasks_per_node': 8, 'partitions': [{'name': 'partition_1', 'nodes': ['node-[001-100]'], 'groups': [{'name': 'group_1', 'nodes': ['node-[001-025]']}, {'name': 'group_2', 'nodes': ['node-[026-050]']}, {'name': 'group_3', 'nodes': ['node-[051-075]']}, {'name': 'group_4', 'nodes': ['node-[076-100]']}]}, {'name': 'partition_2', 'nodes': ['node-[101-200]']}], 'global_env_vars': {'NCCL_IB_GID_INDEX': '3', 'NCCL_IB_TIMEOUT': '20', 'NCCL_IB_QPS_PER_CONNECTION': '4', 'MELLANOX_VISIBLE_DEVICES': '0,3,4,5,6,9,10,11', 'CUDA_VISIBLE_DEVICES': '0,1,2,3,4,5,6,7'}}}
[ERROR] Field 'name': Field required
```